classes/Address.php

```diff
@@ -437,21 +446,28 @@ public static function getCountryAndState($id_address)
      * Specify if an address is already in base.
      *
      * @param int $id_address Address id
+     * @param bool $useCache Use Cache for optimizing queries
      *
      * @return bool The address exists
      */
-    public static function addressExists($id_address)
+    public static function addressExists($id_address, bool $useCache = false)
     {
         if ($id_address <= 0) {
```

Comment:
```suggestion
        if ((int) $id_address <= 0) {
```

Out of the scope but can be cool to have a better chjeck

Reply:
Fixed